docs/mms_on_fargate.md

```diff
@@ -0,0 +1,256 @@
+
+4. As for the security group, it should be either the same security group as you had for the LB, or the one that LB have access to:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.2+SG+Use+existing.png)
+
+5.  Now we can connect our future service to the existing LB. Select the application load balancer and set the LB name:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.3+Add+load+balancing.png)
+
+6. Now we need to specify which port on the LB our service should be listening on:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.4+Configure+load+blancer.png)
+
+7. We are not going to use service discovery now, so uncheck it:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.2.5+Next.png)
+
+8. Also for the sake of the simplicity we are not going to use auto-scaling functionality
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/3.3+Auto+scaling.png)
+
+9. Now we are done-done-done and we can move to the final chapter of our journey:
+
+## Test The Inference
+
+First let's figure out the DNS name of our LB. It should be in EC2 => Load Balancers and click on the LB:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/lb_dns.png)
+
+Now we can run the health checks to verify that it is working:
+```bash
+➜ ~ curl InfraLb-1624382880.us-east-1.elb.amazonaws.com/ping 
+
+http://infralb-1624382880.us-east-1.elb.amazonaws.com/ping
+{
+    "health": "healthy!"
+}
+```
+And now we are finally ready to run our inference! Let's download example iamge:
+```bash
+curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+```
+
+The image:
+
+![](https://s3.amazonaws.com/mms-github-assets/MMS+with+Fargate+Article/kitten.jpg)
+
+And the actual inference:
+```bash
+➜  ~ curl -X POST InfraLb-1624382880.us-east-1.elb.amazonaws.com/squeezenet/predict -F "data=@kitten.jpg"
+{
+  "prediction": [
+    [
+      {
+        "class": "n02124075 Egyptian cat",
+        "probability": 0.8515275120735168
+      },
+      {
+        "class": "n02123045 tabby, tabby cat",
+        "probability": 0.09674164652824402
+      },
+      {
+        "class": "n02123159 tiger cat",
+        "probability": 0.03909163549542427
+      },
+      {
+        "class": "n02128385 leopard, Panthera pardus",
+        "probability": 0.006105933338403702
+      },
+      {
+        "class": "n02127052 lynx, catamount",
+        "probability": 0.003104303264990449
+      }
+    ]
+  ]
+}
+```
+## Instead of a Conclusion
+
+There are many things that we have not touched that can be very useful, like:
+
+
+* setting up integration between MMS and CloudWatch
+* setting auto scaling funcitonality
+* running A/B testing of different versions of the model with the Fargate Deployment concepts
+* connecting Route 53 domain name
+* etc
+
+Each of the topics above probably requires it is own article, so stay tuned ;)
```

Comment:
Each of the topics require
or
Each topic requires

End the sentence, skip the emoji. :) Or period then emoji. I'm not aware that emojis have replaced punctuation. Just co-opted it.

Reply:
Done